### DISCUSSION TITLE
Font Style

### DISCUSSION THREAD

**Discussion by zkkuu** [HelixNebulaStudio/RiseOfTheDead#585] [Q&A] [answered]
What font style does the game use for things like menu tabs, item descriptions, and so on?
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/171686799/2f584de5-de5b-475a-b7e7-3e314bff1612)

**Comment by rafaderren** [comment]
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/76096781/be06aed6-dbf9-4a65-94fb-e9f22b0ee7bd)
i also wanna know this one too

**Comment by MXKhronos** [comment, marked as answer]
Player name is using `Merriweather`, the rest are `Arimo` bolded.
